src/components/QuickStart/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import CodeBlock from '@theme/CodeBlock'
+
+export default function QuickStart({ esm, cjs }) {
+  return (
+    <>
+      <h1>Quick start</h1>
+      <p>Get fastify with NPM:</p>
+
+      <CodeBlock language="bash">npm install fastify</CodeBlock>
```

Comment:
I would add this too:
https://docusaurus.io/docs/2.2.0/markdown-features/code-blocks#npm2yarn-remark-plugin

Reply:
I will look into it

Reply:
npm2yarn plugin in docusaurus works with markdown, I think it doesn't work CodeBlock react component

Reply:
Let's add it to the "nice to have" features